docs/release-notes/data-releases.md

```diff
@@ -6,6 +6,21 @@ platform_key: DOCS_DATA_RELEASES
 
 An ARGO data release is a curated data set of clinical and molecular data submitted to the ARGO Data Platform. Data releases happen approximately four times a year and are cumulative in nature. Released data can be browsed using the [File Repository](https://platform.icgc-argo.org/repository) and [downloaded](/docs/data-access/data-download) using a client tool, provided that access to controlled data has been granted. To access controlled data, please see the [DACO application process here](/docs/data-access/daco/applying).
 
+## Data Release 6.0
+
+**Release Date:** April 03, 2023
+
+#### New Updates
+
+ICGC ARGO is excited to announce the first release of a **new** RNA-Seq analysis workflow: [RNA Seq Alignment](/docs/analysis-workflows/rna-alignment), available now for transcriptome sequencing data from pancreatic cancer(APGI-AU). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU. While clinical data is being actively submitted by the program, a selection of its RNA-Seq data has been reprocessed against the latest GRCh38 Human Reference Genome using the ARGO RNA Seq Alignment.
```

Comment:
Maybe need a space between "pancreatic cancer" and "(APGI-AU)"?

Reply:
Agreed.